docs-build/deploy.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+mkdir ./tmpdocclone
+cd ./tmpdocclone
+git clone https://github.com/${GH_ORG}/docs.git
+
+cd docs
+git checkout -b test${TRAVIS_BUILD_NUMBER}
+
+cp ../../build/cli-commands.md ./docs/using-appsody/cli-commands.md
+
+git add docs/using-appsody/cli-commands.md
```

Comment:
Will there always be changes? Or can we somehow detect if there are changes and skip the rest of the script if not.

Reply:
More is required here than just a diff due to the inclusion of date/time in the appsody docs command generation.